test/kubeobjects/pod/execute.go

```diff
@@ -25,15 +26,31 @@ type ExecutionResult struct {
 type ExecutionQuery struct {
 	pod       v1.Pod
 	command   string
+	arguments []string
 	container string
+	tty       bool
 }
 
-func NewExecutionQuery(pod v1.Pod, container string, command string) ExecutionQuery {
-	return ExecutionQuery{
+func NewExecutionQuery(pod v1.Pod, container string, command string, arguments ...string) ExecutionQuery {
+	query := ExecutionQuery{
 		pod:       pod,
 		command:   command,
 		container: container,
+		arguments: make([]string, 0),
+		tty:       false,
 	}
+
+	for _, arg := range arguments {
+		parts := strings.Split(arg, " ")
+		query.arguments = append(query.arguments, parts...)
```

Comment:
What is argument has (escaped) space in it?
Arguments should be passed as they were provided, splitting is not expected by caller imho.